src/test/java/seedu/address/ui/TestUi/TestPanelLauncher.java

```diff
@@ -0,0 +1,10 @@
+//package seedu.address.ui.TestUi;
```

Comment:
I believe this file should be deleted (its all commented out)? Could you confirm.